App.js

```diff
@@ -71,9 +72,12 @@ const PersistedApp = () => {
   return (
     <PersistGate loading={null} persistor={persistor}>
       <ErrorBoundary>
-        {!!instance && (
-          <Routes routingInstrumentation={routingInstrumentation} />
-        )}
+        <SafeAreaProvider>
+          <StatusBar barStyle="light-content" backgroundColor="black" />
+          {!!instance && (
+            <Routes routingInstrumentation={routingInstrumentation} />
+          )}
+        </SafeAreaProvider>
```

Comment:
Can you check if this have problems with <Container>? Bcz StatusBar was inside of it, maybe we have to delete one.